supportedBackends/jes/src/test/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActorSpec.scala

```diff
@@ -121,6 +121,8 @@ class JesAsyncBackendJobExecutionActorSpec extends TestKit(ActorSystem("JesAsync
     val handle = JesPendingExecutionHandle(jobDescriptor, Seq.empty, run, None)
 
     class TestableJesJobExecutionActor extends JesAsyncBackendJobExecutionActor(jobDescriptor, configurationDescriptor, promise) {
+      override def serviceRegistryActor = system.actorOf(Props.empty)
```

Comment:
Absolutely. I'll put a comment. Basically, when the JES tests are running, there is no `engine`. And, currently, our services are actually implemented in the engine, so we can't spin them up in specs to provide them to a backend implementation. 🙍 

So far the existing tests aren't using the services, so we got that goin' for us, which is nice.